mk64.ld

```diff
@@ -656,13 +656,11 @@ SECTIONS
       #else
       BUILD_DIR/data/sound_data/sequences.o(.data);
       #endif
+      . = ALIGN(0x40);
```

Comment:
Shouldn't there be an ifdef here for US?

I presumed 0x10 for US and 0x30 for EU. surprised eu is 0x40.

Reply:
This matches US after removing the padding inside sequences